programs/ssl/ssl_test_lib.h

```diff
@@ -65,14 +78,16 @@
 #include "mbedtls/ssl.h"
 #include "mbedtls/entropy.h"
 #include "mbedtls/ctr_drbg.h"
+#include "mbedtls/hmac_drbg.h"
+#include "test/certs.h"
```

Comment:
Incorrect conflict resolution. `test/certs.h` is already present a few lines further down. It should have been removed in the merge commit, consistent with the history of the branches: removal of `library/certs.h` in 3 and addition of `hmac_drbg.h` in 2.